README.md

```diff
@@ -156,7 +156,27 @@ class CreateUser
 end
 ```
 
-You'll get an `ArgumentError` if you pass any params not defined in the `inputs_for` block, or if you don't supply all required params. This is completely opt-in so if you don't provide an `inputs_for` block, no checking is performed.
+### Specifying types and nullable fields
+
+You can now also optionally specify types and nullability for your inputs. For the `type:` option, any ruby class can be used.
+
+```ruby
+inputs_for(:call) do
+  required :name, type: String, null: false
```

Comment:
That sounds like the worst kind of compromise.

Creating a library is all about making (hopefully well documented) choices. If you try to account for everything, you will defeat, what I believe to be, your own purpose of keeping it clean and clear.